pom.xml

```
@@ -1,65 +1,41 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>org.jvnet.hudson.plugins</groupId>
-    <artifactId>plugin</artifactId>
-    <version>1.366</version><!-- which version of Hudson is this plugin built against? -->
-    <relativePath>../pom.xml</relativePath>
-  </parent>
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.jenkins-ci.plugins</groupId>
+		<artifactId>plugin</artifactId>
+		<version>4.75</version>
+		<relativePath/>
+	</parent>
+	<properties>
+		<jenkins.version>2.387.3</jenkins.version>

```

The ["Update Jenkins version"](https://www.jenkins.io/doc/developer/tutorial-improve/update-base-jenkins-version/) section of the ["Improve a plugin tutorial"](https://www.jenkins.io/doc/developer/tutorial-improve/) recommends that maintainers follow the guidelines in ["Choosing a Jenkins Version"](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/).  Those guidelines recommend either 2.462.3 or 2.479.3 as good choices.  Jenkins 2.462.3 as a baseline requires Java 11 and supports Java 17 and Java 21.  Jenkins 2.479.3 as a baseline requires Java 17 and supports Java 21.
```suggestion
		<jenkins.version>2.387.3</jenkins.version>

```